lms/services/d2l_api/client.py

```diff
@@ -0,0 +1,46 @@
+from marshmallow import EXCLUDE, fields
+
+from lms.validation._base import RequestsResponseSchema
+
+
+class D2LGroupSetsSchema(RequestsResponseSchema):
+    many = True
+
+    class Meta:
+        unknown = EXCLUDE
+
+    id = fields.Int(required=True, data_key="GroupCategoryId")
+    name = fields.Str(required=True, data_key="Name")
+
+
+class D2LAPIClient:
+    def __init__(self, basic_client, request):
+        self._api = basic_client
```

Comment:
We could expose this directly and let external callers use it directly? This would avoid having to duplicate the `get_token` and `refresh_access_token` methods. As currently they don't change it in anyway.

I know this breaks the one level rule or whatever it's called, but I'm a mad rebel.